src/taco-cli/cli/utils/platformPluginCommandBase.ts

```diff
@@ -146,6 +153,9 @@ export class PlatformPluginCommandBase extends commands.TacoCommandBase {
             return cordovaWrapper.invokePlatformPluginCommand(self.name, self.cordovaCommandParams, data);
         })
             .then(function (): Q.Promise<any> {
+            return self.postInvokeStep(data);
```

Comment:
can we do something like this in run module? 
    public run(data: commands.ICommandData): Q.Promise<ICommandTelemetryProperties> {
        if (data.options.remain == "--list") {
            // run list command logic 
        } else {
        return super.run(data);
        }
    }